src/btree/bt_ret.c

```diff
@@ -194,25 +195,16 @@ __wt_value_return_buf(WT_CURSOR_BTREE *cbt, WT_REF *ref, WT_ITEM *buf, WT_TIME_W
 
     /*
      * WT_PAGE_COL_FIX: Take the value from the original page.
-     *
-     * FIXME-WT-6126: Should also check visibility here
      */
-    if (tw != NULL)
-        WT_TIME_WINDOW_INIT(tw);
+    if (tw != NULL) {
+        __wt_col_fix_fetch_time_window(session, ref, cbt->recno, tw, &found);
```

Comment:
I called it "fetch" because it's doing work (not just retrieving a value from a structure somewhere, which is what we mostly think of when we see "get"). Can change it, or we can try to think of another verb; maybe "lookup" or "retrieve"?

The reason I didn't clear it on not-found is that this avoids a dead store of it for __wt_txn_update_check, which is a hot path, and because it's not inlined there the compiler can't get rid of it.

The reason it doesn't return bool is that it was following __wt_read_cell_time_window. There's no particular reason that one doesn't return bool either. Can change both, or leave it; let me know...

Reply:
The name is changed (per other occurrence below).

Reply:
* I am mildly inclined to leave the time-window init in `__wt_value_return_buf()` because that's what all the other access methods do.
* I am mildly inclined to have both `__wt_col_fix_get_time_window()` and `__wt_read_cell_time_window()` return a boolean "TW found".
* I would move `__wt_col_fix_get_time_window()` to `bt_ret.c`, it only has the one caller, no reason to put it in an inline file, make it static in `bt_ret.c`.

Reply:
I've moved __wt_col_fix_get_time_window and changed both return types.